cfme/tests/intelligence/test_chargeback_assignments.py

```diff
@@ -14,124 +12,122 @@
 
 
 @pytest.mark.meta(blockers=[1273654])
-def test_assign_compute_enterprise(appliance, virtualcenter_provider):
+def test_assign_compute_enterprise(appliance, assigns_collection, virtualcenter_provider):
     view = navigate_to(appliance.server, 'Chargeback')
 
-    enterprise = cb.Assign(
-        assign_to="The Enterprise",
-        selections={
+    enterprise = assigns_collection.instantiate(assign_to="The Enterprise", selections={
             'Enterprise': {'Rate': 'Default'}
         })
     enterprise.computeassign()
 
     # Assert that the selection made is listed on the UI
-    assign_view = view.browser.create_view(AssignmentsView)
+    assign_view = view.browser.create_view(navigator.get_class(assigns_collection, "Compute").VIEW)
```

Comment:
'Compute' and 'Storage' destinations are registered against the entity class, not the collection class, and these get_class calls won't find any match.